pvmismatch/contrib/ModuleMismatchSimulator.py

```diff
@@ -0,0 +1,510 @@
+'''
```

Comment:
Yeah, that was wrapped in the 'typo fix' commit. Sorry, should have identified that change as well. Working on the rest, but need to take care of something else first.